tests/core/input.cddl

```diff
@@ -18,6 +18,11 @@ bar = {
 
 plain = (d: #6.23(uint), e: tagged_text)
 outer = [a: uint, b: plain, c: "some text"]
+plain_arrays = [
+; this is not supported right now. When single-element arrays are supported remove this.
+;	single: [plain],
```

Comment:
we get around this in `babbage.cddl` by creating another truct like:
```
single_plain = [plain]
plain_arrays = [
    single: single_plain,
    ...
]
```

so it's not super high priority to support this, but it would be good if we did when there are less high priority things. I made an issue here: #121